README.md

```diff
@@ -121,6 +121,146 @@ var me = $rdf.sym('http://www.w3.org/People/Berners-Lee/card#i');
 var friend = store.any(me, FOAF('knows'))
 ```
 
+## Running SPARQL queries in the store
+
+RDFLib uses SPARQL queries in order to query data from your local store. Users can find out more information about writing sparql queries 
+
+[here]: https://www.w3.org/TR/rdf-sparql-query/
+
+.
+
+The first step in querying data storerd in your Solid Pod, is to fetch the data from your Pod into your local store. This is performed by the function "loadFromUrl". This function takes in a url from where your Pod can be reached and the variable that defines your local store. 
```

Comment:
`The first step in querying data storerd in your Solid Pod, is to fetch`
->
`When querying data stored in your Solid Pod, the first step is to fetch`